tests/lit-tests/2689.ispc

```diff
@@ -0,0 +1,12 @@
+// RUN: %{ispc} --target=host --nowrap --nostdlib -O2 --emit-llvm-text %s -o - | FileCheck %s
```

Comment:
Host is ok as a target here, but make sure it works on AVX512 targets as well.

Reply:
I have added explicit AVX512 runs.

Reply:
Also `REQUIRES: X86_ENABLED`